test/Makefile

```diff
@@ -143,6 +143,9 @@ test_tools=$(RESULTS_DIR)/d_do_test$(EXE) $(RESULTS_DIR)/sanitize_json$(EXE)
 $(RESULTS_DIR)/%.out: % $(RESULTS_DIR)/.created $(test_tools) $(DMD)
 	$(QUIET) $(RESULTS_DIR)/d_do_test $<
 
+$(RESULTS_DIR)/%.out: % $(RESULTS_DIR)/.created $(RESULTS_DIR)/unit_test_runner$(EXE) $(DMD)
+	$(QUIET) $(RESULTS_DIR)/d_do_test $<
```

Comment:
You duplicated the target here (aka overwrite the target two lines above).

Reply:
Is this target not needed? Seems to work without it.

Reply:
Fixed (I think).